internal/testing/testdata/testdata.go

```diff
@@ -700,6 +700,420 @@ var (
 				Justification:  isDepJustifyContainedByJustification,
 			},
 		},
+		{
+			Pkg: &model.PkgInputSpec{
```

Comment:
is there a reason to duplicate this information if it is already being captured by the original purl? Meaning I understand the need for the `pkgEqual` but seems like it's redundant information to do `isDependency`. We could traverse `pkgEqual` to reach a similar conclusion for the "CPE Purls". Let me know if I am misunderstanding this.

Reply:
You're right, this information is duplicated because the same information could be retrieved using the `isDependency` + `pkgEqual` graph traversal you mentioned.

Let's say, it would be "shortcut" in the graph from CPE purl to dependency but I'm happy to get rid of it to avoid duplication and "normalize" the underlying graph.

Reply:
@pxp928 sorry, I've checked the code since I didn't touch the `IsDependencyIngest` management in this PR.

It looks to me the way it works, it's the way that currently guac expects it to work, i.e. in
https://github.com/guacsec/guac/blob/67371d73d815e57973e983e532d27173e5599558/pkg/ingestor/parser/spdx/parse_spdx.go#L270-L286

`getPackageElement` is meant to retrieve an array of `PkgInputSpec` (not just one) and then iterates over `foundPackNodes`.

Do you think `PkgInputSpec` from CPEs should be filtered out in `getPackageElement` func?